models/company.model.ts

```diff
@@ -0,0 +1,52 @@
+import mongoose, { Model, Schema } from "mongoose";
+import { ICompany } from "./interface/company.model.interface";
+import { v4 } from 'uuid';
+
+const CompanySchema: Schema<ICompany> = new mongoose.Schema({

```

Comment:
again in this model, there should be more meaningful fields as I mentioned earlier in account model like status, zip code, email etc etc should be there. You can do some R&D that normally what fields should be in company entity in multi-tenant system.